.github/workflows/dependencies/dependencies_mac.sh

```diff
@@ -11,4 +11,5 @@ brew update
 brew install boost
 brew install libomp
 brew install pybind11
+brew install llvm
```

Comment:
For OpenMP support on macOS, you don't need the whole of `llvm` - just the subproject `libomp` is enough

Reply:
Thanks! I've just removed it!